app/components/Form/Field.js

```diff
@@ -56,7 +59,14 @@ export function createField(Component: ComponentType<*>) {
     const { error, touched } = meta;
     const hasError = showErrors && touched && error && error.length > 0;
     return (
-      <div className={cx(styles.field, fieldClassName)} style={fieldStyle}>
+      <div
+        className={cx(
+          styles.field,
+          fieldClassName,
+          radioOrCheckbox && styles.radioOrCheckbox
```

Comment:
Couldn't you make `CheckBox.Field` always pass this prop as the `fieldClassName` instead?
Something like:

```js
const RawField = createField(CheckBox);
CheckBox.Field = ({ fieldClassName, ...props }: FieldProps) => (
  <RawField fieldClassName={cx(fieldClassName, styles.radioOrCheckbox)} {...props} />
);
```

Would be nice to avoid adding a boolean prop to `createField`.

Reply:
I might have misunderstood, but I thought `fieldClassName` was meant to be a prop the user could use for styling. Ie
```
<Field
    component={CheckBox.Field}
    label="Går det bra?"
    name="fine"
    fieldClassName={styles.myField}
/>
```
As such, I didn't want to override it and remove that option from users. Am I wrong?

Reply:
Yes, but if you look at the code above you'll see that it still takes in `fieldClassName`, it just appends on the `styles.radioOrCheckbox` classname to it as well.

Reply:
i.e. `fieldClassName={cx(fieldClassName, styles.radioOrCheckbox)}`